sdk/src/main/java/com/qonversion/android/sdk/internal/di/DependencyInjection.kt

```diff
@@ -0,0 +1,20 @@
+package com.qonversion.android.sdk.internal.di
+
+import android.app.Application
+import com.qonversion.android.sdk.internal.di.controllers.ControllersAssembly
+import com.qonversion.android.sdk.internal.di.controllers.ControllersAssemblyImpl
+import com.qonversion.android.sdk.internal.di.misc.MiscAssembly
+import com.qonversion.android.sdk.internal.di.misc.MiscAssemblyImpl
+
+import com.qonversion.android.sdk.internal.di.services.ServicesAssembly
+import com.qonversion.android.sdk.internal.di.services.ServicesAssemblyImpl
+
+object DependencyInjection : ControllersAssembly by ControllersAssemblyImpl,
+    ServicesAssembly by ServicesAssemblyImpl, MiscAssembly by MiscAssemblyImpl {
+
+    override fun init(application: Application) {
```

Comment:
I'd offer to rename all the init methods to `initialize` as `init` is the kotlin reserved name and we can have collisions in the future